README.md

```diff
@@ -1,50 +1,56 @@
-# Welcome to your Expo app 👋
+# Typography Component
 
-This is an [Expo](https://expo.dev) project created with [`create-expo-app`](https://www.npmjs.com/package/create-expo-app).
+The `Typography` component provides a consistent, theme-aware way to render text throughout the app. It supports a wide range of text types (headings, body, labels, etc.), sizes, and weights, with built-in support for light/dark themes and a default font family.
 
-## Get started
+## Purpose
 
-1. Install dependencies
+- Standardize text styles across the app
+- Support light and dark themes automatically
+- Encourage consistent use of font sizes, weights, and colors
 
-   ```bash
-   npm install
-   ```
+## Usage
 
-2. Start the app
+```tsx
+import { Typography } from './components/Typography';
 
-   ```bash
-   npx expo start
-   ```
-
-In the output, you'll find options to open the app in a
-
-- [development build](https://docs.expo.dev/develop/development-builds/introduction/)
-- [Android emulator](https://docs.expo.dev/workflow/android-studio-emulator/)
-- [iOS simulator](https://docs.expo.dev/workflow/ios-simulator/)
-- [Expo Go](https://expo.dev/go), a limited sandbox for trying out app development with Expo
+<Typography type="title" size="XLarge" weight="Bold">Extra Large Bold Title</Typography>
+<Typography type="body" size="Medium" weight="Regular">Medium Regular Body Text</Typography>
```

Comment:
Why have you edited the main readme file just for Typegraphy component?
don't change it, and if you want to write documentation for Typegraphy, use separate file under docs.